package.json

```diff
@@ -68,6 +68,7 @@
     ]
   },
   "resolutions": {
-    "@nuxt/kit": "^3.4.3"
```

Comment:
I don't remember why this was needed on the first place but couldn't we just remove it now?

Reply:
you could. but sometimes renovate/dependabot end up installing multiple versions of these on upgrade prs and this helps make the upgrades smoother. your call though.

Reply:
I see, thanks for the info!